### DISCUSSION TITLE
Feedback Request : Menu Redesign

### DISCUSSION THREAD

**Discussion by GeekInTheNorth** [GeekInTheNorth/Stott.Security.Optimizely#229] [Ideas]
I am considering replacing the menus within Stott Security to utilize the built in menu functionality within the Optimizely CMS and would appreciate some feedback from the user base.  

## New Menu Design:

By utilizing the built in Optimizely menu framework, the same navigation can be provided for the security Add-On while maintaining a structure and design that is consistent with other administration menus within the CMS.  In the scenario below I have added CSS style overrides to shrink the width of the menu by 100 pixels to ensure that there is no loss in usable horizontal real estate while also offsetting height gain with a title for clarity.

![image](https://github.com/user-attachments/assets/664dd996-b378-47e6-a075-f8be186f83a6)

## Existing Menu Design:

In the existing menu I am using a horizontal tab interface which results in 3 separate menu systems, the top module selector, the left hand menu and then the Stott Security menu.

![image](https://github.com/user-attachments/assets/e9fec0f6-6df3-49fb-8522-14c12d6b02ae)

**Comment by scottreed** [comment]
I'm not using it currently but have recommended it at Niteco. Personally I prefer the Optimizely menu system as it keep consistency with other addons using it and a standard UI/UX

**Comment by GeekInTheNorth** [comment]
I've finished the conversion to use the standard Optimizely menus.  I've opted to keep the current title bar and to combine the AddOn title with the section title.

![image](https://github.com/user-attachments/assets/4fd80bd5-0043-44d2-a2ab-326f7f910a1c)

**Comment by GeekInTheNorth** [comment]
This has now been delivered in Release 2.8, see #244
